docs/UserGuide.adoc

```diff
@@ -728,27 +723,28 @@ To correct this, you have to enter a valid `INDEX` that is not more than the num
 [[view-c-command]]
 === View a client profile: `view-c` --- Li Zi Ying
 
-You might want to view the complete information of a certain client in your client list. This complete information includes the client's full details, their exercise table and personal best table. `view-c INDEX` shows all available information of the client at the specified index.
+You might want to view the complete information of a certain client in your client list. `view-c INDEX` shows all available information of the client at the specified index.
 
 *Format*: `view-c INDEX`
 
 ==== Parameters
-This section acts as a summary of the important things to note when using `view-c` including their parameters.
+This section shows the parameters of `view-c` and the important things to note about them:
 
 [options='header']
 [cols="15%,85%"]
 |====================
-| Parameters | Important points to note
-| `INDEX` + |
-`INDEX` refers to the index number shown in the displayed client list +
+| Parameter | Important points to note
+| `INDEX` + a|
+* `INDEX` refers to the index number shown in the displayed client list +
 
-`INDEX` must be a positive integer (ie. 1, 2, 3, ...) +
+* `INDEX` must be a positive integer (ie. 1, 2, 3, ...) +
 
+* `INDEX` is compulsory +
```

Comment:
I am unsure if we are still adding "compulsory" statements

Reply:
Agreed, no need to repeat again.